src/services/generateS3ObjectPrefixes.spec.ts

```diff
@@ -123,19 +123,19 @@ describe('object prefixes', () => {
       'firehose/2022/11/22/00'
     ]
 
-    const result = generateS3ObjectPrefixes('2022/11/21', '2022/11/21')
+    const result = generateS3ObjectPrefixes('2022-11-21', '2022-11-21')
     expect(result).toEqual(expectedResult)
   })
 
   test('invalid date string', () => {
     expect(() => {
       generateS3ObjectPrefixes('invalid', 'invalid')
-    }).toThrow('String not valid date')
+    }).toThrow("String 'invalid' is not a valid date")
```

Comment:
This will fail the pipeline won't it? I think Linting will want single quotes.

Reply:
Having checked, looks like Prettier is ok with this due to the quotes within quotes!